src/stores/api/api.store.tsx

```diff
@@ -8,22 +8,34 @@ export interface APIModel {
   classes: Record<string, any> | null;
   class: Record<string, any> | null;
   abilities: Record<string, any> | null;
+  races: Record<string, any> | null;
+  race: Record<string, any> | null;
```

Comment:
I wonder what is the value type for this Record ❓

Reply:
All of these are objects. What should I change these to?